check.js

```diff
@@ -27,26 +34,63 @@ function dubslevel(postid) {
     return 0;
 }
 function dubs(post) {
```

Comment:
dubs/trips/quads etc share so much code now that you should make a function to keep it simple, then just pass in highlight length, color, and element.